scaffold/tugboat/steps/1-init.sh.twig

```diff
@@ -49,12 +49,16 @@ docker-php-ext-install gd
 # of GD. Lullabot will likely be publishing an ADR recommending it too.
 apt-get install -y imagemagick
 {% if memory_cache_type == "memcached" %}
-# Install the PHP memcache extension. The Drupal Redis module recommends
-# phpredis so no PHP extension is required.
+# Install the PHP memcache extension.
 
 apt-get install -y zlib1g-dev
-pecl install memcache
+yes '' | pecl install -f memcache
```

Comment:
Interesting, I've never needed to do any of this in tugboat or CI - just `pecl install <something>`. What's up here?

Reply:
it should run fine in non-interactive mode, but won't run without asking questions if you put that directly into the tugboat terminal - bit hard for me to test at the moment with our setup so just left it at what I can test directly